<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatwindow</class>
 <widget class="QMainWindow" name="chatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>921</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(245, 245, 245)</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>903</width>
       <height>643</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>641</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/contact.png) 0 0 0 0 stretch stretch;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="chatList">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>481</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(234, 232, 231)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="horizontalWidget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="chatWindowTitleLabel">
              <property name="minimumSize">
               <size>
                <width>61</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>418</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>91</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
              </property>
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(231, 231, 231,0.3);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="ui_mag">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>241</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(245, 245, 245)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="sendPhotoButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
             </property>
             <property name="text">
              <string>发送图片</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>36</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="sendFileButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
             </property>
             <property name="text">
              <string>发送文件</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>36</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="receiveFileButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
             </property>
             <property name="text">
              <string>接收文件</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>328</width>
               <height>36</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(231, 231, 231,0.3);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="idt">
           <property name="minimumSize">
            <size>
             <width>589</width>
             <height>227</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(245, 245, 245);
color:black;
font-size:18;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>378</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="voiceInputButton">
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
             </property>
             <property name="text">
              <string>语音识别</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn1">
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(233, 233, 233);
color: rgb(7, 193, 96);</string>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>38</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
